.circleci/config.yml

```diff
@@ -114,12 +118,13 @@ jobs:
             - browser-tools/install-chrome:
                 replace-existing: false
       - run: npm run test -- --browsers=<<parameters.browser>>
+      - run: npm run cov:unit:publish
       - save_cache_cmd:
           node-version: <<parameters.node-version>>
       - store_test_results:
           path: dist/reports/tests/
       - store_artifacts:
-          path: dist/reports/
+          path: coverage
```

Comment:
We should add this folder to the `webpack.dev.js` exclude list so that `test:watch` continues to work.

Reply:
nevermind, thought this was a webpack config..